dashboard/lib/widgets/task_grid.dart

```diff
@@ -71,15 +71,15 @@ class TaskGridContainer extends StatelessWidget {
 /// Results are displayed in a matrix format. Rows are commits and columns
 /// are the results from tasks.
 class TaskGrid extends StatefulWidget {
-  const TaskGrid({
-    Key? key,
-    // TODO(ianh): We really shouldn't take both of these, since buildState exposes status as well;
-    // it's asking for trouble because the tests can (and do) describe a mutually inconsistent state.
-    required this.buildState,
-    required this.commitStatuses,
-    this.filter,
-    this.useAnimatedLoading = false,
-  }) : super(key: key);
+  const TaskGrid(
+      {Key? key,
```

Comment:
You can change this to `super.key`, and remove the call to super below

Reply:
Done.